### PR TITLE
feature: Tinlake rewards claims migration 

### PR DESCRIPTION
**Pull request by NunoAlexandre**
Fork of https://github.com/centrifuge/centrifuge-cli/pull/21
Based off #24 

- We add support to migrate the standalone chain `RadClaims.UploadAccount` and `RadClaims.AccountBalances` to the  parachain `Claims.UploadAccount` and `Claims.ClaimedAmounts`, respectively.

- From #24 make both of the above items verifications handle the renaming and we drop the migration of `RootHashes`.

### CODE REVIEW

File: packages/plugins/migration/src/migration/common.ts [hunk 1]
```diff
@@ -2,7 +2,21 @@ import {xxhashAsHex} from "@polkadot/util-crypto";
...
+    if (map.has(key)) {
+        // We have checked that it exists
+        // @ts-ignore
+        return map.get(key)
+    } else {
+        map.set(key, new Map())
+        // We have checked that it exists
+        // @ts-ignore
+        return map.get(key)
+    }
```
Comment (NunoAlexandre) [inline packages/plugins/migration/src/migration/common.ts:16]:
```suggestion
    if (!map.has(key)) {
        map.set(key, new Map())
    }
        
     // We have checked that it exists
     // @ts-ignore
     return map.get(key)
  }
```

File: packages/plugins/migration/src/migration/migrate.ts [hunk 2]
```diff
@@ -52,11 +60,6 @@ export async function verifyMigration(
             if(failed.length !== 0) {
                 failedVerification.push(...failed);
             }
-        } else if (destKey === xxhashAsHex("Claims", 128) + xxhashAsHex("RootHashes", 128).slice(2)){
```
Comment (mustermeiszer) [inline packages/plugins/migration/src/migration/migrate.ts:55]: Why is this deleted and not the upload account? Will we migrate the upload account?

**Review by mustermeiszer — APPROVED** [review]
LGTM!

File: packages/plugins/migration/src/migration/migrate.ts (hunk 2 above)
Comment (NunoAlexandre) [reply packages/plugins/migration/src/migration/migrate.ts:55]: Yes, we want to migrate the upload account. The reason to delete the RootHashes is that we can't know which entry is the latest so we don't want to migrate everything, which would mean we would be migrating dust data essentially.
